msbuild: update to 16.0+xamarinxplat.2019.04.08.19.19

KexyBiscuit opened the pull request.

KexyBiscuit committed:
msbuild: update to 16.0+xamarinxplat.2018.09.26.17.53

MingcongBai commented on extra-mono/msbuild/autobuild/build:
```diff
@@ -1,16 +1,8 @@
-
+./install-mono-prefix.sh "$PKGDIR"/usr/bin/
+mkdir -p "$PKGDIR"/usr/share/doc/"$PKGNAME"
+install -Dm644 "$SRCDIR"/LICENCE "$PKGDIR"/usr/share/doc/"$PKGNAME"
```
You dont need the last two lines - Autobuild3 will do this automatically if it finds "$SRCDIR"/COPYING or "$SRCDIR"/LICENCE.

KexyBiscuit replied:
I love that.

KexyBiscuit committed:
msbuild: work around the environment variable VERSION

KexyBiscuit committed:
msbuild: no need for copying license

KexyBiscuit commented:
Blocked by Microsoft/msbuild#4196, mono/mono#13602.

KexyBiscuit committed:
msbuild: update to 16.0+xamarinxplat.2019.04.08.19.19

KexyBiscuit committed:
msbuild: change source tarball uri, add file hash

KexyBiscuit committed:
msbuild: adopt scripts and patches from mono

  - https://github.com/mono/linux-packaging-msbuild/tree/1bc2ba019ec0fc563f4ae3a14e94e9d884b6562d

KexyBiscuit committed:
mono: oops, we're not Debian
